serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+
+<Message type="tip">
+
+If you previously had multiple namespaces to separate your SQS resources, we recommend that you use [Scaleway Projects](/console/my-project/concepts/#project) with the new version to handle this. By creating multiple Projects, activating SQS in each one, and then creating SQS credentials in each Project, you will be able to achieve separation of resources, and credentials scoped to the SQS broker of each Project.
+
+</Message>
+
+You can recreate your credentials and queues either via the [console](https://console.scaleway.com/) (once the new version is in production from the time of GA) or via the new `v1beta1` API directly. In terms of the API, the [current `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version will continue to be available 30 days after migration/General Availability, co-existing with the new `v1beta1` version. The documentation for both will be accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api) via a dropdown at the top of the Messaging and Queuing API page. 
+
+Note the following important information about the migration/crossover period, when both versions of the API will exist:
+
+**Before GA**
+- Only the `v1alpha1` API is available. All your resources exist in this API.
+- The console is based on this API, so when you view your resources in the console, you are viewing resources in the `v1alpha1` API.
+
+**At the time of GA**
+- The `v1alpha1` API is still available. Your resources continue to exist in this API until you delete them.
+- The `v1beta1` API also becomes available. You should recreate your resources (credentials and queues) in this API.
+- The console now uses the `v1beta1` API. **You will no longer be able to see or manage your old resources from the `v1alpha1` API via the console**. Any resources now created via the console exist in the `v1beta1` API.
+- Messaging and Queuing with SQS is now billable. Whether you are using the old version or the new one, via the API or other devtools or the console, you are now billed for the messages you send.
+
+**30 days after GA**
+- The `v1alpha1` API is decommissioned. Any resources still existing on this API are deleted.
+
+**The URL/endpoints for your Messaging and Queuing SQS service and for your queues will not be the same in the two versions**. Make sure that when you recreate your resources with the new version, you remember to update the endpoint in any code, scripts, applications and architectures that were using  the old endpoint. If you do not do so, they will continue to access the "old" resources rather than the new ones you have created for the new version.
```

Comment:
```suggestion
**The URL/endpoints for your Messaging and Queuing SQS service and for your queues will not be the same in the two versions**. Make sure that when you recreate your resources with the new version, you remember to update the endpoint in any code, scripts, applications and architectures that were using the old endpoint. If you do not do so, they will continue to access the "old" resources rather than the new ones you have created for the new version.
```